.codecov.yml

```diff
@@ -0,0 +1,6 @@
+comment: true
+coverage:
+  status:
+    project:
+      default:
+        informational: true
```

Comment:
This whole file might not be needed any longer if https://community-release.mattermost.com/core/pl/ay8eip538jnnmdzjfizxzrc77c get addressed, but I wanted to kick things off.